lib/tasks/client.rake

```diff
@@ -14,6 +14,17 @@ namespace :client do
     Client.__elasticsearch__.refresh_index!
   end
 
+  desc "Create Alias"
```

Comment:
I don't understand why we need an alias for the `clients` index? Indexing takes less than 5 min. Is it for zero downtime?

Reply:
Yeah, well I thought that once I was at it, I could put it to all indexes.

Reply:
That is a funny argument :) Why do we do aliasing and how does it help with `clients` and `providers`?

Reply:
well I mean the change is indeed for aiming to `zero downtime`. But you are right, it would only be needed if the indexes growth considerably.

Reply:
Which means that you remove the rake task or keep it in?

Reply:
I can remove them